### PR TITLE
[BCHDCC-38] - Accessibility Fixes: 1.3.1 Info and Relationships

### PR DESCRIPTION
**Pull request by emmabjj**
## Summary
What is this branch for what does it do? 

[Implement accessibility fixes for criterion 1.3.1 info and relationships.](https://www.w3.org/TR/2007/WD-UNDERSTANDING-WCAG20-20071211/content-structure-separation-programmatic.html)

As a user whom requires adaptive technology (i.e. screen reader, braiille display, etc.), I want to navigate a website with correct structure, labels, and relationships, so I can access the information using my adaptive technology. 

**Zube Card Referenced** 00

**File Addition(s)** 
- `whatever.rb`: A whatever file to add whatever config

**Files Modified**
- `settings.yml`: Added 'English' to the list of languages

**Libraries Added**
- `gem 'devise', '~> 4.7'`: Used for auth set up

### Tests to Run
- `spec/features/whatver_spec.rb`: Tests the function `hello_world`

### TODO
- [ ] Add UI Tests for Service Areas
- [ ] Possibly add model validation for areas

### CODE REVIEW

File: app/views/component/detail/_address.html.haml [hunk 1]
```diff
@@ -2,7 +2,8 @@
   %span{ itemscope: '', itemtype: 'http://schema.org/PostalAddress', itemprop: 'address' }
     %section.icon-text-block
       %i.fa.fa-map-marker
-      %h4.annotated-text-block Location
+      %h2.visuallyhidden= "Address"
+      %h3.annotated-text-block Location
```
Comment (anna-dorigo) [inline app/views/component/detail/_address.html.haml:6]: This is correctly mentioned by the screen reader

File: app/views/component/detail/_location_short_desc.html.haml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 - if location.short_desc.present?
   %section.short-desc
-    %h4 ABOUT
+    %h3 ABOUT
```
Comment (anna-dorigo) [inline app/views/component/detail/_location_short_desc.html.haml:3]: This shows correctly on the screen reader

File: app/views/component/detail/_address.html.haml (hunk 1 above)
Comment (emmabjj) [reply app/views/component/detail/_address.html.haml:6]: Would you like me to revert these changes? This is about the page skipping header structure and not about the screen reader per say.

File: app/views/component/locations/results/_map_view.html.haml [hunk 3]
```diff
@@ -2,7 +2,11 @@
   #map-canvas
   = render 'component/search/map_data', search: search
   %noscript
-    -# Hide map when JavaScript is disabled.
+    %p.map-alternative
+      = "Map is not available because JavaScript is disabled. Please use the address below for directions."
+      %br
+      %span{ itemprop: 'streetAddress' }><
+        = superscript_ordinals(street_address_for(location.address))
```
Comment (anna-dorigo) [inline app/views/component/locations/results/_map_view.html.haml]: This line doesn't let me perform a search:
image

Comment (anna-dorigo) [reply app/views/component/locations/results/_map_view.html.haml]: When I comment it out I can perform a search but I see some CSS issues on the search menu: 
image

Comment (emmabjj) [reply app/views/component/locations/results/_map_view.html.haml]: Okay, let me fix that! I'm glad you caught that - thanks!

**Review by anna-dorigo — APPROVED** [review]
Works great, the issue is gone and the home page search reads correctly on the screen reader.  All forms reads correctly the labels. Text is consistent with the header hierarchy.  Feel free to merge (I think there is only a small conflict with the develop branch)

**Pull request merged**